code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/ContentCardEventObserver.kt

```diff
@@ -29,7 +34,7 @@ class ContentCardEventObserver(private var callback: ContentCardCallback?) : Aep
 /**
  * Interface to handle different callback events which can occur for a displayed content card.
  */
-interface ContentCardCallback {
+interface ContentCardUIEventListening {
```

Comment:
spoorthi mentioned that she thought it would be good to match the name that the iOS implementation used.

Reply:
i can revert the name change if needed

Reply:
Haven't seen java/kotlin libraries use continuous tense for interfaces/classes. So my preference is to use the 
`ContentCardUIEventListener` if the aim to so make them similar instead of matching.

Reply:
sounds good, i'l update